src/Components/PeoplePage/PeoplePage.tsx

```diff
@@ -0,0 +1,29 @@
+import React, { useEffect, useState } from 'react';
+import { Person } from '../../types/Person';
+import { getPeople } from '../../api/people';
+import { PeopleTable } from '../PeopleTable';
+
+export const PeoplePage: React.FC = React.memo(() => {
+  const [people, setPeople] = useState<Person[]>([]);
+
+  useEffect(() => {
+    const fetchPeoples = async () => {
+      const response = await getPeople();
+
+      setPeople(response);
+    };
+
+    try {
+      fetchPeoples();
+    } catch (error) {
+      throw new Error(`${error}`);
+    }
```

Comment:
you could also use `catch` for promises, by doing so you avoid leaving promise returned by `fetchedPeoples()` just «in the air»
```
fetchPeople()
  .catch(error => console.log(error)); // or something like that
```
https://devtrium.com/posts/async-functions-useeffect